web/themes/custom/novel/templates/layout/node--event--full.html.twig

```diff
@@ -20,6 +20,16 @@
       </div>
     </div>
     <dl class="list-description event-description__info list-description--event">
+      {% if content.field_event_place.0 %}
+      <div class="list-description__item">
+        <dt class="list-description__key">
+          {{ "Place" | trans }}
```

Comment:
You need to add a translation for this.

web/profiles/dpl_cms/translations/da.po